autopush/tests/test_main.py

```diff
@@ -45,6 +45,66 @@ def test_resolve_host_no_interface(self, mock_socket):
         eq_(ip, "google.com")
 
 
+class SettingsAsyncTestCase(trialtest.TestCase):
+    def test_update_rotating_tables(self):
+        from autopush.db import create_rotating_message_table, get_month
+        # Create the rotating table so the test passes
+        create_rotating_message_table()
+        settings = AutopushSettings(
+            hostname="google.com", resolve_hostname=True)
```

Comment:
while popular, probably want to use "example.com". There's a lot of optimization to ensure example.com resolves.